src/pre_defined_classes/mathematical_program/medium/q2/HeronTest.java

```diff
@@ -0,0 +1,18 @@
+package pre_defined_classes.mathematical_program.medium.q2;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.StringLiteral;
+
+public class HeronTest extends BaseTest {
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The area of the triangle is: 8.317902379807064 units squared.")
```

Comment:
With questions like this we should still check the math with a test - the test should work even if the question was not Parsons